examples/passing/DctorName.purs

```diff
@@ -0,0 +1,11 @@
+module Main where
+
+import Control.Monad.Eff.Console (log)
+
+newtype Bar' = Bar' Int
+
+data Foo' = Foo' Bar'
+
+data Baz'' = Baz''
+
+main = log "Done"
```

Comment:
Could you please add an example where you `case` on one of these new constructors? Thanks.